### ISSUE TITLE
`FakeQuerySet.values_list` returns list of objects instead of list of pks

### ISSUE THREAD

**Issue by m1kola** [wagtail/django-modelcluster#57]
If model have relations, `FakeQuerySet.values_list` returns [list of related objects](https://github.com/torchbox/django-modelcluster/blob/v2.0/modelcluster/queryset.py#L123) instead of list of primary keys.

See Django's [`QuerySet.values_list`](https://docs.djangoproject.com/en/1.9/ref/models/querysets/#values-list) documentation.

**Comment by gasman** [comment]
An illustration of this, using wagtaildemo (putting this here because the Django docs are a bit confusing on this point):

```
>>> from demo.models import HomePage, HomePageCarouselItem
>>> from wagtail.wagtailimages.models import Image
>>> hp = HomePage.objects.get()
>>> hp.carousel_items.values_list('image')  # real queryset
<QuerySet [(1001,), (12,), (11,)]>
>>> new_hp = HomePage(carousel_items=[HomePageCarouselItem(image=Image.objects.first()), HomePageCarouselItem(image=Image.objects.first())])
>>> new_hp.carousel_items.values_list('image')  # fake queryset
[(<Image: Wagtail by Mark Harkin>,), (<Image: Wagtail by Mark Harkin>,)]
```

**Comment by matt-dalton** [comment]
Are there any updates (or workarounds) for this? We have a use case that where this is making life tricky

**Comment by gasman** [comment]
Fixed in #149.
